backend/tests/lorapatch.py

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2017-2018, Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+import os
+import sys
+import pathlib
+
+
+def ensure_lora_patched():
+    try:
+        # patch db_structure in case we have oio_rest/oio_common installed
+        import oio_common  # noqa F401
+        from oio_common import db_structure
+        orgfile = pathlib.Path(db_structure.__file__)
+        topdir = pathlib.Path(__file__).absolute().parent.parent.parent
+        patchfile = topdir / "setup" / "db_structure.py"
+
+        if orgfile.read_text() != patchfile.read_text():
+            orgfile.write_text(patchfile.read_text())
```

Comment:
This does not belong in a separate module, but in `utils.py`, and should most certainly not happen as a side-effect of importing said module. Overwriting a Python source file is evil enough as-is, but doing so at import-time is even worse.

Still: You shouldn't need this. If you do, let's talk about how to adjust LoRA so you don't.

Reply:
This will have a short life. Pretty soon there will be integration data in all objects in lora. Using copious amounts of hours to fix something that will be gone in a split second seems a little over the top. But if You can get it to work without this, be my guest.